File: Exercicio-10.java
import java.util.Scanner;
import java.text.DecimalFormat; 

public class Main
{
  public static void main (String[]args)
  {
    Scanner sc = new Scanner (System.in);
    DecimalFormat df = new DecimalFormat("#,###.00");
    double salary, tax, total;

    System.out.printf("Digite o seu salário: ");
    salary = sc.nextDouble();
  
    tax = (salary * 5 / 100);
    total = salary - tax;
  
    System.out.println("Salário Bruto: " + df.format(salary));
    System.out.println("Imposto de Renda: " + df.format(tax));
    System.out.println("Salário Liquido: " + df.format(total));
  }
}

File: Exercicio-10.c
#include <stdio.h>

int main(void) {

  float salary, tax, total;

  printf("Digite o seu salário: ");
  scanf("%f", &salary);
  
  tax = (salary * 5 / 100);
  total = salary - tax;
  
  printf("\nSalário Bruto: %.2f \n", salary);
  printf("Imposto de Renda: %.2f \n", tax);
  printf("Salário Liquido: %.2f \n", total);
  
  return 0;
}
